doc/changes/8806.md

```diff
@@ -0,0 +1 @@
+- Fix `dune rpc status` on Windows (#8806, fixes #8799, @nojb)
```

Comment:
This fixes a little more than `status`. It should fix all `dune rpc` commands in fact.

Reply:
Looks like `monitor` too.

Reply:
Thanks for the review. I amended the Changes entry.